reconcile/cli.py

```diff
@@ -166,9 +166,10 @@ def run_integration(func, *args):
         sys.exit(1)
 
 
-def init_log_level(log_level):
+def init_log_level(log_level, invoked_subcommand):
     level = getattr(logging, log_level) if log_level else logging.INFO
-    logging.basicConfig(format='%(levelname)s: %(message)s', level=level)
+    log_format = f'[{invoked_subcommand}] %(levelname)s: %(message)s'
```

Comment:
f-strings are Python >= 3.6. Not sure how big of a problem that is for this project, but according to the `setup.py`, we do support previous version of Python.

Reply:
thanks for spotting that! removed the py3 line.